test/integration/integration.ts

```diff
@@ -1,20 +1,10 @@
+/* eslint-disable mocha/no-hooks-for-single-case -- Keeps test setup consistent */
```

Comment:
Starting a thread here so we can discuss. 
@mvadari Which requests is this PR missing? (Going off of [this documentation](https://xrpl.org/transaction-methods.html), sign & sign_for are not included, submit is covered by individual transactions, tx_history is deprecated, and we don't seem to have tx_entry) - Are there other requests that should be added here that aren't included on this list?

Reply:
I change my mind, I think we do need a specific `submit` test. @natenichols is there a specific reason we didn't add a `transaction_entry` request?

Reply:
I chatted with Nathan about this, and AFAIK the reason is because we suggest people use tx instead of tx_entry in the documentation: https://xrpl.org/transaction_entry.html 
@natenichols Chime in if there's a different reason I'm forgetting!